loom/common/platform/platformFile.h

```diff
@@ -99,13 +108,12 @@ int platform_dirExists(const char *path);
 int platform_removeDir(const char *path);
 
 /*!
- * Normalizes a path to use the system folder delimiter
+ * Normalizes a path to use the system folder delimiter.
+ * This function modifies the buffer in-place.
  *
- * @param path the full path to the directory to remove
- * @return a string with the normalized path (note that this is a shared string buffer
- * and only valid between calls)
+ * @param path Path to normalize.
  */
-const char *platform_normalizePath(const char *path);
+void platform_normalizePath(const char *path);
```

Comment:
Not at this point, no. It just changes slashes. If we need more later, we can change it.